composer.json

```diff
@@ -12,18 +12,18 @@
         }
     ],
     "require": {
-        "php": ">=5.3.0",
+        "php": "^7.2.5",
         "psr/container": "^1.0"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "^3.2"
+        "symfony/phpunit-bridge": "^3.4|^4.4|^5.0"
```

Comment:
I would even go with `^5.0` only here. No need to use older versions of the bridge to run tests.